src/utils/datetime.cljs

```diff
@@ -36,6 +38,22 @@
    11 "nov"
    12 "dec"})
 
+(def ^:const full-months
+  "Maps the corresponding string representation of a month
+   By it's numeric index as in cljs-time"
+  {1  "january"
```

Comment:
how is it used? will it be translated?

Reply:
yes i added translation related to month and weekdays into `translations/en.json` and use it like this 
`(i18n/label (get dt/full-months month)`

Reply:
@flexsurfer, @mohsen-ghafouri

I remember a past PR where I mentioned the same thing, I think it's relevant. I don't think we should be adding these translations manually for weekdays, months, etc. We should be leveraging the Google library that's already localized, that's the whole point of having it, unless I'm missing something obvious.

For example:

```clojure
(def weekday-fmt (get-formatter-fn (constantly "E")))

(.format ^js (weekday-fmt) (t/now))
;; => "Wed"
```

Reply:
I unresolved the thread @mohsen-ghafouri. I think this is an open point. I don't think we should add these manual translations, it looks wrong. The Google library can give us the names of weekdays and months based on a locale.

Reply:
@ilmotta  sorry i didn't see this one, so this function will generate day names like 'Wed,' and I should display weekdays in 2 characters. Will using substring cause any issues with translations in different languages?

Reply:
i tried different date formatting and none of them provide a symbol for two-letter weekday abbreviations, and using substring could potentially cause issues, because the length of the abbreviated weekday names can vary between different languages and locales.

Reply:
You are totally correct @mohsen-ghafouri, if the requirements are that we need two letter weekdays, then unfortunately this is not available as far as I checked the library. Thanks for trying the approach I proposed.